test/verify.test.js

```diff
@@ -33,3 +33,19 @@ test('Throw SemanticReleaseError if "changelogFile" option is a whitespace Strin
   t.is(error.name, 'SemanticReleaseError');
   t.is(error.code, 'EINVALIDCHANGELOGFILE');
 });
+
+test('Throw SemanticReleaseError if "changelogTitle" option is not a String', t => {
+  const changelogTitle = 42;
+  const [error] = t.throws(() => verify({changelogTitle}));
+
+  t.is(error.name, 'SemanticReleaseError');
+  t.is(error.code, 'EINVALIDCHANGELOGTITLE');
+});
+
+test('Throw SemanticReleaseError if "changelogTitle" option is a whitespace String', t => {
+  const changelogTitle = '  \n \r ';
+  const [error] = t.throws(() => verify({changelogTitle}));
+
+  t.is(error.name, 'SemanticReleaseError');
+  t.is(error.code, 'EINVALIDCHANGELOGTITLE');
+});
```

Comment:
Can you add a test similar to `Throw SemanticReleaseError if "changelogFile" option is a whitespace String` for consistency reasons?

Reply:
You commented right under that test :)

Reply:
Oh right sorry. I meant a test similar to `Throw SemanticReleaseError if "changelogFile" option is an empty String`. The reason is that an empty String is falsy, while a white space one is not.

Reply:
Done